### PR TITLE
Update ESLint configuration and fix lint issues

### LINKED ISSUE #1: Enable ESLint
Fix lint errors and enable ESLint.

### PR DESCRIPTION
**Pull request by jianjunz**
Fix #1.

### CODE REVIEW

**Comment by jianjunz** [comment]
@taste1981, please take a look. I cannot add you as a reviewer.

File: src/.eslintrc.json [hunk 1]
```diff
@@ -1,25 +1,16 @@
...
-    "indent": ["error", 2, {
-      "SwitchCase": 1
-    }],
-    "eol-last": "error",
-    "spaced-comment": ["warn", "always"],
-    "space-before-blocks": ["error", "always"],
-    "space-infix-ops": "error"
+    "max-len": ["error", {
+      // This line is part of the MIT license.
+      "ignorePattern": "The above copyright notice and this permission notice shall be included in all"
```
Comment (taste1981) [inline src/.eslintrc.json]: The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

Comment (taste1981) [reply src/.eslintrc.json]: OK for matching purpose only.

Comment (jianjunz) [reply src/.eslintrc.json]: Only this line exceeds 80 characters.

**Pull request merged**